src/components/Section.js

```diff
@@ -14,4 +13,8 @@ export default class Section {
   addItem(item) {
     this._renderer(item);
   }
+
```

Comment:
* New cards should be added only with `prepend` to the start of the section otherwise the user doesn’t see the new card and may think that it hasn’t been created


Please, create `prependItem` and `appendItem` inside `Section`. They should add cards into the container **inside** the class rather than outside it